viam-orb-slam3_test.go

```diff
@@ -164,36 +162,11 @@ func setupDeps(attr *slamConfig.AttrConfig) registry.Dependencies {
 	for _, sensor := range attr.Sensors {
 		cam := &inject.Camera{}
 		switch sensor {
-		case "good_lidar":
-			cam.NextPointCloudFunc = func(ctx context.Context) (pointcloud.PointCloud, error) {
-				return pointcloud.New(), nil
-			}
-			cam.StreamFunc = func(ctx context.Context, errHandlers ...gostream.ErrorHandler) (gostream.VideoStream, error) {
-				return nil, errors.New("lidar not camera")
-			}
-			cam.ProjectorFunc = func(ctx context.Context) (transform.Projector, error) {
-				return nil, transform.NewNoIntrinsicsError("")
-			}
-			cam.PropertiesFunc = func(ctx context.Context) (camera.Properties, error) {
-				return camera.Properties{}, nil
-			}
-			deps[camera.Named(sensor)] = cam
-		case "bad_lidar":
-			cam.NextPointCloudFunc = func(ctx context.Context) (pointcloud.PointCloud, error) {
-				return nil, errors.New("bad_lidar")
-			}
-			cam.StreamFunc = func(ctx context.Context, errHandlers ...gostream.ErrorHandler) (gostream.VideoStream, error) {
-				return nil, errors.New("lidar not camera")
-			}
-			cam.ProjectorFunc = func(ctx context.Context) (transform.Projector, error) {
-				return nil, transform.NewNoIntrinsicsError("")
-			}
-			deps[camera.Named(sensor)] = cam
 		case "good_camera":
 			cam.StreamFunc = func(ctx context.Context, errHandlers ...gostream.ErrorHandler) (gostream.VideoStream, error) {
 				return gostream.NewEmbeddedVideoStreamFromReader(
 					gostream.VideoReaderFunc(func(ctx context.Context) (image.Image, func(), error) {
-						return image.NewNRGBA(image.Rect(0, 0, 1024, 1024)), nil, nil
+						return image.NewRGBA(image.Rect(0, 0, 1024, 1024)), nil, nil
```

Comment:
[q] Why did this change?

Reply:
It was changed to get another test to pass however I realize now that it is no longer necessary to have this change. Fixed.